app.js

```diff
@@ -2,6 +2,7 @@ require("dotenv").config();
 const { createClient } = require("redis");
 const express = require("express");
 const cors = require("cors");
+const e = require("express");
 
 const redisClient = createClient({
   host: process.env.REDIS_HOST,
```

Comment:
You'll need to use the `url` parameter here instead of `host` and `port` to make it work in docker.